content/post/ml_talks_moscow.md

```diff
@@ -0,0 +1,159 @@
+---
+author:
+date: 2017-06-07
+title: "About our tech talks in Moscow"
+image: /post/ml_talks_moscow/back_audience.jpg
+description: "On June, 3-2017, we dedicated our regular [source{d} tech talks](http://talks.sourced.tech/) to Machine Learning and we chose to host the event in Moscow. For this conference, we invited speakers from Russia and abroad and gathered about 80 neural network aficionados in an former industrial area of the city. Let's make now a brief follow-up of this day."
+categories: ["technical"]
+---
+
+<style>
+p.dt {
+  margin-top: -16px;
+  font-style: italic;
+}
+.twitter-tweet {
+  margin-left: auto;
+  margin-right: auto;
+}
+.verbatim {
+  font-size: 0.85em;
+}
+</style>
+
+![picture of the audience during Egor's speech](/post/ml_talks_moscow/audience.jpg)
+
+On June, 3-2017, we dedicated our regular [source{d} tech talks](http://talks.sourced.tech/) to Machine Learning and we chose to host the event in Moscow. For this conference, we invited speakers from Russia and abroad and gathered about 80 neural network aficionados in an former industrial area of the city.
+
+# Day's programme
+
+To begin with, everybody joined together around a hearty welcome breakfast inside [KL10CH](https://www.linkedin.com/company/kl10ch) spaces in th city center.
```

Comment:
Right, I could put the russian link http://kl10.ch/ Anyway this information would target Russian people.